docs/guides/saturn-0/whats-new.mdx

```diff
@@ -7,42 +7,38 @@ serve as an introduction to some new and exciting changes to the Rocket Pool pro
-This upgrade consists parameter changes, [Houston Hotfix (RPIP-63)](https://snapshot.org/#/rocketpool-dao.eth/proposal/0x1d75c5247cf457d26fbf7f26622c21de912565256fc3273b92a40aa14dd1af3d)
-and [Tokenomics Rework Prelude (RPIP-62)](https://snapshot.org/#/rocketpool-dao.eth/proposal/0x7f6c978251e0ae752c619cc459331e5b3cc081aed13b133159b43fb6194b3304).
+This upgrade consists of the changes detailed in [Tokenomics Rework Prelude (RPIP-62)](https://snapshot.org/#/rocketpool-dao.eth/proposal/0x7f6c978251e0ae752c619cc459331e5b3cc081aed13b133159b43fb6194b3304).
 :::
 
 
 ## ETH only minipools in Saturn 0 
 
-- The RPL requirement for new minipools are a thing of the past! Minipools launched after the inception of Saturn 0 do not require RPL stake to be deployed. 
-Node operators can now create minipools with an upfront cost of 8 ETH. In addition, a node operator may stake RPL to qualify for additional rewards 
-and access voting power within the [Protocol DAO](guides/houston/pdao#the-protocol-dao-pdao). 
+- The RPL requirement for new minipools is a thing of the past! Minipools launched after the inception of Saturn 0 do not require RPL stake to be deployed.
+Node operators can now create minipools with an upfront cost of 8 ETH, which will feature a commission ranging from 10% to 14%. In addition, a node
+operator may optionally stake RPL to qualify for additional rewards and access voting power within the [Protocol DAO](guides/houston/pdao#the-protocol-dao-pdao). 
 
 - Nodes with an RPL stake of any size qualify for RPL issuance rewards. This unlocks RPL rewards for Node Operators under the 10% collateral cliff 
 who were previously unqualified for RPL issuance rewards. 
 
 - This means nodes with an RPL stake of 0-15% of borrowed ETH receive the maximum APY on RPL. Nodes still earn RPL yield beyond 15% collateral, at a decreasing rate.
 Feel free to take a look how RPL issuance beyond 15% borrowed ETH is defined in [RPIP-30](https://rpips.rocketpool.net/RPIPs/RPIP-30#specification).
 
-- Saturn 0 minipools have a 5% base ETH comission, which will be temporarily boosted up to 14% after [Reward Tree Spec v10](https://rpips.rocketpool.net/assets/rpip-62/rewards-calculation-spec), up until the launch of Saturn 1.
-
 ## New Commission Structure (Saturn 0 part 2)
 ::: tip NOTE
 Dynamic Commission will be enabled when v10 of the [Reward Tree Spec](https://rpips.rocketpool.net/assets/rpip-62/rewards-calculation-spec) is implemented. It's currently in development, so keep an eye on [#protocol](https://discord.com/channels/405159462932971535/405163979141545995) 
 announcements in our [Discord Server](https://discord.com/invite/rocketpool) for updates!
 :::
 
-We're excited to announce that newly launched Saturn 0 minipools will feature a commission structure ranging from 5% to 14%.
+We're excited to announce that newly launched Saturn 0 minipools will feature a commission structure ranging from 10% to 14%.
```

Comment:
We have this message in the SN as 5-14: https://github.com/rocket-pool/smartnode/pull/681#event-14740168889

I reckon we should just keep it consistent with the SN?

Reply:
I responded in discord: https://discord.com/channels/405159462932971535/922648971351056475/1299160800203902986